From 6a8bfc52981ba92abeb4820cfe76b3c4427e1a57 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 25 Jul 2009 19:25:13 +0200
Subject: [PATCH] apr: force some MSys-specific settings

With MSys, there is a winsock2.h, but we do not want to use it.  Force
that, and also avoid misdetecting an absence of in_addr.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 apr/include/apr.h.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/apr/include/apr.h.in b/apr/include/apr.h.in
index fc26dc2..b6e0aeb 100644
--- a/apr/include/apr.h.in
+++ b/apr/include/apr.h.in
@@ -106,7 +106,7 @@
 #define APR_HAVE_TIME_H          @timeh@
 #define APR_HAVE_UNISTD_H        @unistdh@
 #define APR_HAVE_WINDOWS_H       @windowsh@
-#define APR_HAVE_WINSOCK2_H      @winsock2h@
+#define APR_HAVE_WINSOCK2_H      0
 
 /** @} */
 /** @} */
@@ -204,7 +204,7 @@ extern "C" {
 
 #define APR_HAVE_CORKABLE_TCP   @have_corkable_tcp@ 
 #define APR_HAVE_GETRLIMIT      @have_getrlimit@
-#define APR_HAVE_IN_ADDR        @have_in_addr@
+#define APR_HAVE_IN_ADDR        1
 #define APR_HAVE_INET_ADDR      @have_inet_addr@
 #define APR_HAVE_INET_NETWORK   @have_inet_network@
 #define APR_HAVE_IPV6           @have_ipv6@
-- 
1.6.3.2.1299.gee46c

